### DISCUSSION TITLE
Exclude Flag Release

### DISCUSSION THREAD

**Discussion by bieri** [leonardochaia/dotnet-affected#83] [General]
Last public release appears to be 3.1.1 on March 9. Was hoping to get the exclude feature into a release (committed on June 15). Any idea when the next version will be released?

**Comment by leonardochaia** [comment]
Hi @bieri . I was hoping on getting more features in before release but my time is limited.

[v3.2.0](https://github.com/leonardochaia/dotnet-affected/releases/tag/v3.2.0) has been released. Packages should be uploaded in a few minutes.

Leo

**Comment by bieri** [comment]
Thanks so much for the reply and publishing. The outstanding features are "Affected Test output via traversal" ? I may have a use case for something like that and can try and take a look in the coming days

**Reply by leonardochaia** [reply]
Hi @bieri , the next feature I am planning on tackling is #81 .

Basically, I have an idea of [how I wanted implemented](https://github.com/leonardochaia/dotnet-affected/issues/81#issuecomment-1594569802). I started implementation and discovered several changes to shared components are needed. I may have some time this weekend to wrap it up.
